WebSite/Site.Master

```diff
@@ -33,6 +33,7 @@
                   <li><a href="csrf.aspx"> CSRF</a></li>
                   <li><a href="Deserialization.aspx"> Deserializattion</a></li>
                   <li><a href="idor.aspx"> IDOR</a></li>
+                  <li><a href="Misc.aspx"> MISC</a></li>
```

Comment:
Is 'MISC' the best name for this? When people look at the menu, they're not going to know what Misc is referring to.